Project-1/README.md

```diff
@@ -12,63 +12,296 @@ Tip #4: The work we do is challenging; that should be assumed. You are smart eno
+            ¬¬Pa                     (5)
+            ¬¬Qa                     (5)
+             Qa                      (7)
+             Pa                      (6)
+  
+  ¬Pa (4)              ¬Qa (4)
+   x                   x
+             
   (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  Tree proof:
+  
+   ¬∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))  
+   ¬∀y(¬(Pa ∨ Qa) → (¬Pa ∧ ¬Qa))     (1)
+    ¬(¬(Pa ∨ Qa) → (¬Pa ∧ ¬Qa))      (2)
+           ¬(Pa ∨ Qa)                (3)
+           ¬(¬Pa ∧ ¬Qa)              (3)
+	       ¬Pa                   (4)
+               ¬Qa                   (4)
+
+¬¬Pa (5)                ¬¬Qa (5)  
+  Pa (8)                  Qa (9)
+  x                       x
+  
   (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
-  (d) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
-```
+  Tree proof:
+  ¬∀x∀y((¬Px ∨ ¬Qx) →  ¬(Px ∧ Qx))
+   ¬∀y(¬(Pa ∨ Qa) →  ¬(Pa ∧ Qa))     (1)
+    ¬(¬(Pa ∨ Qa) →  ¬(Pa ∧ Qa))      (2)
+           ¬Pa ∨  ¬Qa               (3)
+           ¬¬(Pa ∧ Qa)              (3)
+	    Pa ∧ Qa                 (5)
+	       Pa                   (6)
+               Qa                   (6)
+
+  ¬Pa (4)                 ¬Qa (4)  
+  x                        x
+   
+  
+  (d) ∀x∀y(¬(Px ∨ Qx) → ¬(Px ∧ Qx))
+ Tree proof: same as (c)
+ 
 	
 9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
 
+
+  first step: (¬p ∨ ¬q) → ¬(p ∧ q)
+   
+ 1. ¬p ∨ ¬q          premise
+|2. ¬¬(p ∧ q)        assumption
+|3. p ∧ q            ¬E2
+|4. p                ∧E3
+|5. q                ∧E3
+|6. ⊥                ⊥I 1,4,5
+ 7. ¬¬¬(p ∧ q).      ¬I 2-6
+ 8. ¬(p ∧ q)
+ 
+See the attachment: image 1-2.
+
+
+  Second step: ¬(p ∧ q) → ¬p ∨ ¬q
+ 
+ 1.  ¬(p ∧ q)               premise
+|2.  ¬(¬p ∨ ¬q)             assumption
+| |3.  ¬p                   assumption 
+| |4.  ¬p ∨ ¬q              ∨I3
+| |5. ⊥                    ⊥I2,4
+|6. p                      ⊥I3-5
+| |7. ¬q                    assmuption
+| |8. ¬p ∨ ¬q               ∨I7
+| |9. ⊥                    ⊥I2,8
+|10. q                     ⊥I 7-9
+|11. p ∧ q                  ∧I6,10
+|12. ⊥                     ⊥I1,11
+13. ¬¬(¬p ∨ ¬q)             ⊥I2-12         
+14. ¬p ∨ ¬q                 ¬E13
+
+See the attachment: image 1-2.
+
+Third step:  ¬(p v q) → ¬p  ∧ ¬q
+
+See the attachment: image 3.
+
+Fourth step:  ¬p ∧ ¬q → ¬(p v q)
+
+see the attachment: image 4.
+
+
 10. Compare and contrast the proofs provided for (a) in your answers to questions 8 and 9. Explain the different assumptions, strategies, etc. exhibited in tree proofs vs natural deduction proofs. 
 
+For (a), the natural deduction proof applies mutiple deductive rules including disjunction elimination or double negation elimination (DNE), while the Tree proof only uses indirect derivation. That is a tree proof just focuses on finding possible logical contradictions among negated subformulas of the target formula.
```

Comment:
Tree proofs use deductive rules too, such as conjunction elimination and DeMorgan's laws! The main difference is natural deduction allows for more proof methods, e.g. conditional, than tableau which only uses indirect.